static/src/js/actions/focusedCollection.js

```diff
@@ -180,130 +184,155 @@ export const getFocusedCollection = () => async (dispatch, getState) => {
         }
       }
     }`
+  try {
+    const response = await graphQlRequestObject.search(graphQuery, {
+      params: {
+        conceptId: focusedCollectionId,
+        includeHasGranules: true,
+        includeTags: defaultCmrSearchTags.join(',')
```

Comment:
You don't need to have the tags be part of the subsequent variable requests, those just add more metadata but, you aren't parsing any of it in your `retrieveVariablesRequest` functions

Reply:
which ones would you like me to remove?

Reply:
Oh I just mean that when you pass that parameter: `includeTags ` to graphql/CMR you are asking to, alongside the standard metadata, also get the `tag` concepts associated to that collection. If you aren't parsing those tag concepts then you don't need to request them